### DISCUSSION TITLE
Can this be applied to online game chat?

### DISCUSSION THREAD

**Discussion by Root-FTW** [openai/whisper#572] [Ideas]
Can this be applied to gaming?

Example:

I speak only Spanish, I play Fortnite every day after work, my friends are from the US and only speak Spanish, I have been learning a little English for a few months and my friends a little Spanish, but that's not enough.

It is possible to do something that captures the audio in real time that my PC is generating (my friends speaking in English) and generates the subtitles in my Spanish language, or simply in English (I can read English perfectly).

THANKS

**Comment by Ar-Kareem** [comment]
For starters I think you should try the demo here https://huggingface.co/spaces/Amrrs/openai-whisper-live-transcribe

**Comment by ethanzrd** [comment]
Hey, a bit late. Have you managed to do this? This could be a great project to work on.

**Comment by UmangRajpara13** [comment]
I am thinking of adding such an integration to this [project](https://github.com/voyagingstar/able) where other apps can ask for transcription support from an instance of whisper model currently running on the GPU. i call it "shared automatic speech recognition". so far it can execute user defined actions( CLI, Automation scripts) which are linked to voice commands in a JSON dictionary and also it can be used to controls desktops app that support integration of Extensions/Plugins/Add-ons (example: VS Code) and other desktop app that do not support such integration (by adding small piece of code in source code of the app)
